decidim-proposals/config/locales/es.yml

```diff
@@ -28,6 +29,10 @@ es:
             proposal_answering_enabled: Respuesta oficial a propuestas habilitadas
             proposal_edit_before_minutes: Las propuestas pueden ser editadas por los autores antes de que pasen estos minutos
             proposal_limit: Límite de propuestas por usuario
+            proposal_wizard_step_1_help_text: Texto de ayuda del primer paso del asistente de creación de propuestas
```

Comment:
We shouldn't add other locales than `en` here. All the translations are managed at Crowdin.